src/app/plugins/suggestions.plugin.ts

```diff
@@ -0,0 +1,25 @@
+import { MessageEmbed} from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import Constants from '../../common/constants';
+
+export default class SuggestionPlugin extends Plugin {
+  public commandName: string = 'suggest';
+  public name: string = 'Suggestions Plugin';
+  public description: string = 'Create suggestions for lion to output';
+  public usage: string = 'suggest <string>';
+  public override pluginChannelName: string = Constants.Channels.Public.ServerRequests;
+  public permission: ChannelType = ChannelType.All;
+  private _embed: MessageEmbed = new MessageEmbed();
```

Comment:
I dont see a reason to store the embed as a property of the class when nothing is carried over from each use.

You could add a footer to the embed to justify this, but even then `new MessageEmbed` is not an expensive operation.
I would just make a new one each time